### PR TITLE
[REM] crm: google spreadsheet integration

### PR DESCRIPTION
**Pull request by jero-odoo**
The information in this document is no longer relevant as of 16. Google Drive and Google Spreadsheet integrations are no longer supported. I added a redirect to existing documentation describing the features of Odoo Spreadsheets, including the CRM specific templates. 

Only a tech review needed, no content added.

### CODE REVIEW

**Comment by ksc-odoo** [comment]
> The information in this document is no longer relevant as of 16. Google Drive and Google Spreadsheet integrations are no longer supported. I added a redirect to existing documentation describing the features of Odoo Spreadsheets, including the CRM specific templates.
> 
> Only a tech review needed, no content added.

Gotcha! Looks good to me!

**Review by ksc-odoo — APPROVED** [review]
Understand this only needs a tech-review. Nicely done! @jero-odoo

File: redirects/16.0.txt
Comment (samueljlieber) [inline redirects/16.0.txt:58]: Small change to follow the existing comment formatting, and lets move to the top of this section to maintain alphabetical order 🙂
```suggestion
applications/sales/crm/performance/google_spreadsheets.rst applications/finance/spreadsheet/templates.rst                             # /google_spreadsheets --> /spreadsheet/templates
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @jero-odoo nice job with this removal PR! I have a very small change that I will push up after this review, and then I will approve if all checks pass 🙂

**Review by samueljlieber — APPROVED** [review]
LGTM! @StraubCreative

**Review by StraubCreative — APPROVED** [review]
lgtm 👍 
on to you @odoo/user-docs-structure-review

**Comment by xpl-odoo** [comment]
👍 
@robodoo r+

**Pull request closed without merging**
